README.md

```diff
@@ -18,57 +20,42 @@ To hash a list of bytes, invoke the [`convert`][convert] method on the
 objects.
 
 ```dart
-import 'package:crypto/crypto.dart';
 import 'dart:convert'; // for the UTF8.encode method
```

Comment:
I don't think this comment is necessary.

Reply:
ack